src/pages/who-we-are/_DerivNumbers.tsx

```diff
@@ -125,7 +125,7 @@ const DerivNumbers = () => {
                 >
                     {deriv_numbers.map((number) =>
                         number.map(({ count, title }) => (
-                            <Flex key={title.props.translate_text} fd="column" height="unset">
+                            <Flex key={title.props.translate_text || title} fd="column" height="unset">
```

Comment:
the `title` in deriv_numbers is a `ReactElement` using it as key prop is not suitable please check it out, thanks.

Reply:
@michio-rework don't have any other options, we have title and count as a `ReactElement ` and index. No more options

Reply:
@eduard-deriv just for my understanding, What is the reason behind changing this?
can we use `count.props.translate_text` instead? (if there is an issue with the key)

Reply:
@niloo-fs we can use `count.props.translate_text` or `title.props.translate_text` , both works the same and correctly. I solved dublicated errors. I expect exactly what Michio meant.

Reply:
thanks @niloo-fs @michio-rework 👍